src/app/components/SelectField.tsx

```diff
@@ -2,11 +2,11 @@ import * as React from 'react'
 import classnames from 'classnames'
 
 type ListProps = {
-  list: any[]
+  list: { name: string | number }[]
   placeholder: string
   setValid?: React.Dispatch<React.SetStateAction<boolean>>
-  value: any
-  setValue?: any
+  value: string | undefined
+  setValue?: React.Dispatch<React.SetStateAction<string | undefined>>
```

Comment:
What do you think about renaming `setValue` to `onChange`. Like in original input element

Reply:
I think "setValue" will be better, because component does not receive an event handler, but a function for changing the state.